### ISSUE TITLE
pyGenomeTracks & Epilogo's for a >8kb region

### ISSUE THREAD

**Issue by favilaco** [deeptools/pyGenomeTracks#51]
Hi,

I'm having issues when trying to plot more than 8kb regions.

This is my input.qcat

```
chr10	12829801	12830001	id:1174730,qcat:[ [-0.2403,25], [-0.004582,17], [0,13], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,16], [0,14], [0,1], [0,12], [0,11], [0,10], [0,9], [0,6], [0,5], [0,4], [0,3], [0,2], [0.004148,15], [0.009821,18], [0.2731,8], [0.5698,7] ]
chr10	12830001	12830201	id:1174731,qcat:[ [-0.2858,25], [0,13], [0,24], [0,23], [0,22], [0,21], [0,20], [0,18], [0,16], [0,15], [0,1], [0,12], [0,10], [0,9], [0,6], [0,5], [0,4], [0,3], [0,2], [0.006585,17], [0.00834,14], [0.04288,11], [0.1326,19], [0.2277,8], [0.5698,7] ]
chr10	12830201	12830401	id:1174732,qcat:[ [-0.2638,25], [0,13], [0,24], [0,23], [0,22], [0,21], [0,20], [0,16], [0,1], [0,12], [0,10], [0,9], [0,6], [0,5], [0,4], [0,3], [0,2], [0.004148,15], [0.006585,17], [0.00834,14], [0.009821,18], [0.01357,11], [0.0322,7], [0.04267,19], [0.7562,8] ]
chr10	12830401	12830601	id:1174733,qcat:[ [-0.198,25], [0,13], [0,24], [0,23], [0,22], [0,21], [0,20], [0,16], [0,1], [0,12], [0,10], [0,9], [0,6], [0,5], [0,4], [0,3], [0,2], [0.001742,19], [0.00834,14], [0.009821,18], [0.01357,11], [0.0237,17], [0.0322,7], [0.1122,15], [0.4468,8] ]
chr10	12830601	12830801	id:1174734,qcat:[ [-0.1028,25], [0,13], [0,24], [0,23], [0,22], [0,21], [0,20], [0,18], [0,16], [0,1], [0,12], [0,10], [0,9], [0,6], [0,5], [0,4], [0,3], [0,2], [0.001742,19], [0.00834,14], [0.01357,11], [0.0171,7], [0.04988,15], [0.06851,17], [0.2502,8] ]
chr10	12830801	12831001	id:1174735,qcat:[ [-0.0503,25], [0,13], [0,24], [0,23], [0,22], [0,21], [0,20], [0,18], [0,16], [0,15], [0,1], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,2], [0.00834,14], [0.06996,19], [0.09463,17], [0.2731,8] ]
chr10	12831001	12831201	id:1174736,qcat:[ [-0.03942,25], [0,13], [0,24], [0,23], [0,22], [0,21], [0,20], [0,18], [0,16], [0,15], [0,1], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,2], [0.00834,14], [0.09463,17], [0.1001,19], [0.2277,8] ]
chr10	12831201	12831401	id:1174737,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.009821,18], [0.02835,25], [0.09463,17], [0.2058,8] ]
chr10	12831401	12831601	id:1174738,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.009821,18], [0.04005,25], [0.06851,17], [0.2058,8] ]
chr10	12831601	12831801	id:1174739,qcat:[ [-0.004582,17], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.009821,18], [0.125,25], [0.1435,8] ]
chr10	12831801	12832001	id:1174740,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.009821,18], [0.1376,25], [0.1435,8] ]
chr10	12832200	12832400	id:1174741,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.07013,8], [0.202,25] ]
chr10	12832400	12832600	id:1174742,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.07013,8], [0.202,25] ]
chr10	12832600	12832800	id:1174743,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.07013,8], [0.202,25] ]
chr10	12832800	12833000	id:1174744,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.05383,8], [0.2152,25] ]
chr10	12833000	12833200	id:1174745,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.03847,8], [0.2285,25] ]
chr10	12833200	12833400	id:1174746,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.006585,17], [0.01096,8], [0.2285,25] ]
chr10	12833400	12833600	id:1174747,qcat:[ [-0.004582,17], [0,2], [0,23], [0,21], [0,20], [0,18], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.001742,19], [0.01096,8], [0.0367,16], [0.0432,22], [0.07697,24], [0.1002,25] ]
chr10	12833600	12833800	id:1174748,qcat:[ [-0.004582,17], [0,2], [0,23], [0,21], [0,20], [0,18], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.001742,19], [0.01096,8], [0.0367,16], [0.0432,22], [0.07697,24], [0.1002,25] ]
chr10	12833800	12834000	id:1174749,qcat:[ [0,2], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.006585,17], [0.01096,8], [0.07697,24], [0.1503,25] ]
chr10	12834000	12834200	id:1174750,qcat:[ [-0.004582,17], [0,2], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.01096,8], [0.07697,24], [0.1631,25] ]
chr10	12834200	12834400	id:1174751,qcat:[ [0,2], [0,23], [0,22], [0,21], [0,20], [0,18], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.01096,8], [0.0367,16], [0.04267,19], [0.05186,25], [0.06851,17], [0.07697,24] ]
chr10	12834400	12834600	id:1174752,qcat:[ [0,2], [0,23], [0,22], [0,21], [0,20], [0,18], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.01048,16], [0.01096,8], [0.0237,17], [0.04267,19], [0.05379,24], [0.1002,25] ]
chr10	12834600	12834800	id:1174753,qcat:[ [0,2], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.006585,17], [0.01096,8], [0.05379,24], [0.1631,25] ]
chr10	12834800	12835000	id:1174754,qcat:[ [0,2], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.01096,8], [0.0237,17], [0.05379,24], [0.1503,25] ]
chr10	12835000	12835200	id:1174755,qcat:[ [-0.007525,24], [-0.004582,17], [0,2], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.02414,8], [0.2152,25] ]
chr10	12835200	12835400	id:1174756,qcat:[ [-0.007525,24], [-0.004582,17], [0,2], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.02414,8], [0.2152,25] ]
chr10	12835400	12835600	id:1174757,qcat:[ [-0.007525,24], [0,2], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,15], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.02414,8], [0.02835,25], [0.03243,14], [0.3556,17] ]
chr10	12835600	12835800	id:1174758,qcat:[ [-0.2238,25], [-0.01146,8], [-0.007525,24], [0,2], [0,23], [0,21], [0,20], [0,19], [0,18], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.02404,15], [0.0377,13], [0.1601,22], [0.1722,14], [0.4149,16], [0.5085,17] ]
chr10	12835800	12836000	id:1174759,qcat:[ [-0.353,25], [-0.03174,8], [0,2], [0,24], [0,21], [0,20], [0,18], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.004148,15], [0.0112,23], [0.0377,13], [0.1722,14], [0.2158,17], [0.2988,22], [0.6715,16], [0.9577,19] ]
chr10	12836000	12836200	id:1174760,qcat:[ [-0.123,25], [-0.03174,8], [0,2], [0,24], [0,23], [0,21], [0,20], [0,18], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.001742,19], [0.01098,13], [0.02404,15], [0.2131,14], [0.2158,17], [0.2988,22], [0.3663,16] ]
chr10	12836200	12836400	id:1174761,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.004148,15], [0.0367,16], [0.06246,14], [0.1376,25], [0.2158,17] ]
chr10	12836400	12836600	id:1174762,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.3523,25] ]
chr10	12836600	12836800	id:1174763,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.3523,25] ]
chr10	12836800	12837000	id:1174764,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.3523,25] ]
chr10	12837000	12837200	id:1174765,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.3523,25] ]
chr10	12837200	12837400	id:1174766,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.3523,25] ]
chr10	12837400	12837600	id:1174767,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.3523,25] ]
chr10	12837600	12837800	id:1174768,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.3523,25] ]
chr10	12837800	12838000	id:1174769,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.006585,17], [0.3241,25] ]
chr10	12838000	12838200	id:1174770,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.3523,25] ]
chr10	12838200	12838400	id:1174771,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.3523,25] ]
chr10	12838400	12838600	id:1174772,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.3523,25] ]
chr10	12838600	12838800	id:1174773,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.3523,25] ]
chr10	12838800	12839000	id:1174774,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.3523,25] ]
chr10	12839000	12839200	id:1174775,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.0237,17], [0.3102,25] ]
chr10	12839200	12839400	id:1174776,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.004148,15], [0.06851,17], [0.2689,25] ]
chr10	12839400	12839600	id:1174777,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.02404,15], [0.1523,17], [0.2152,25] ]
chr10	12839600	12839800	id:1174778,qcat:[ [-0.03174,8], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,16], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.04988,15], [0.1227,17], [0.2152,25] ]
chr10	12839800	12840000	id:1174779,qcat:[ [-0.2403,25], [0,13], [0,24], [0,23], [0,22], [0,21], [0,20], [0,18], [0,1], [0,12], [0,11], [0,10], [0,9], [0,8], [0,7], [0,6], [0,5], [0,4], [0,3], [0,2], [0.001742,19], [0.0367,16], [0.09635,14], [0.4778,15], [1.16,17] ]
chr10	12840000	12840200	id:1174780,qcat:[ [-0.3627,25], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,18], [0,12], [0,11], [0,10], [0,9], [0,8], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.06246,14], [0.07037,13], [0.1049,16], [0.2403,19], [0.6177,15], [1.348,17] ]
chr10	12840200	12840400	id:1174781,qcat:[ [-0.3832,25], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,18], [0,12], [0,11], [0,10], [0,9], [0,8], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.07037,13], [0.09635,14], [0.2728,16], [0.589,17], [0.7629,19], [0.8637,15] ]
chr10	12840400	12840600	id:1174782,qcat:[ [-0.3191,25], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,12], [0,11], [0,10], [0,9], [0,8], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.03243,14], [0.1023,18], [0.1069,13], [0.1438,16], [0.3556,17], [0.4014,19], [0.9146,15] ]
chr10	12840600	12840800	id:1174783,qcat:[ [-0.198,25], [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,12], [0,11], [0,10], [0,9], [0,8], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.001742,19], [0.03243,14], [0.0367,16], [0.1069,13], [0.1405,18], [0.3556,17], [0.8134,15] ]
chr10	12840800	12841000	id:1174784,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.01096,8], [0.07958,15], [0.202,25] ]
chr10	12841000	12841200	id:1174785,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.05383,8], [0.2152,25] ]
```

my categories_file.json
```
{
"categories":{
"24": [
          "Repressed PolyComb",
          "#808080"
        ],
        "25": [
          "Quiescent/Low",
          "#ffffff"
        ],
        "20": [
          "ZNF genes & repeats",
          "#66cdaa"
        ],
        "21": [
          "Heterochromatin",
          "#8a91d0"
        ],
        "22": [
          "Poised Promoter",
          "#e6b8b7"
        ],
        "23": [
          "Bivalent Promoter",
          "#7030a0"
        ],
        "1": [
          "Active TSS",
          "#ff0000"
        ],
        "3": [
          "Promoter Downstream TSS with DNase",
          "#ff4500"
        ],
        "2": [
          "Promoter Upstream TSS",
          "#ff4500"
        ],
        "5": [
          "Transcription 5",
          "#008000"
        ],
        "4": [
          "Promoter Downstream TSS",
          "#ff4500"
        ],
        "7": [
          "Transcription 3",
          "#008000"
        ],
        "6": [
          "Transcription",
          "#008000"
        ],
        "9": [
          "Transcription Regulatory",
          "#c2e105"
        ],
        "8": [
          "Weak transcription",
          "#009600"
        ],
        "11": [
          "Transcription 3 Enhancer",
          "#c2e105"
        ],
        "10": [
          "Transcription 5 Enhancer",
          "#c2e105"
        ],
        "13": [
          "Active Enhancer 1",
          "#ffc34d"
        ],
        "12": [
          "Transcription Weak Enhancer",
          "#c2e105"
        ],
        "15": [
          "Active Enhancer Flank",
          "#ffc34d"
        ],
        "14": [
          "Active Enhancer 2",
          "#ffc34d"
        ],
        "17": [
          "Weak Enhancer 2",
          "#ffff00"
        ],
        "16": [
          "Weak Enhancer 1",
          "#ffff00"
        ],
        "19": [
          "DNase only",
          "#ffff66"
        ],
        "18": [
          "Enhancer Acetylation Only",
          "#ffff00"
        ]
}
```
and my tracks.ini
```
[x-axis]
#optional
#fontsize=20
# default is bottom meaning below the axis line
# where=top

[spacer]
# height of space in cm (optional)
height = 0.5


[input.qcat]
file=/Users/tmp/input.qcat

# title of track (plotted on the right side)
title = INPUT


categories_file=/Users/favilaco/categories_file.json
```

This command (region > 8kb) doesn't work (I also tried tabix with -m 1000 instead of the default 200, but it doesn't work):
`
pyGenomeTracks --tracks tracks.ini --region chr10:12836000-12841000 -o OUTPUT.pdf
`

LOG:
```
INFO:pygenometracks.tracksClass:time initializing track(s):
INFO:pygenometracks.tracksClass:0.0038521289825439453
DEBUG:pygenometracks.tracksClass:Figure size in cm is 40 x 4.0. Dpi is set to 72

INFO:pygenometracks.tracksClass:plotting 1. [x-axis]
/Users/favilaco/miniconda3/lib/python3.5/site-packages/matplotlib/cbook/__init__.py:424: MatplotlibDeprecationWarning:
Passing one of 'on', 'true', 'off', 'false' as a boolean is deprecated; use an actual boolean (True/False) instead.
  warn_deprecated("2.2", "Passing one of 'on', 'true', 'off', 'false' as a "
INFO:pygenometracks.tracksClass:plotting 2. [spacer]
INFO:pygenometracks.tracksClass:plotting 3. [input.qcat]
Traceback (most recent call last):
  File "/Users/favilaco/miniconda3/bin/pyGenomeTracks", line 11, in <module>
    main(args)
  File "/Users/favilaco/miniconda3/lib/python3.5/site-packages/pygenometracks/plotTracks.py", line 304, in main
    trp.plot(args.outFileName, *region, title=args.title)
  File "/Users/favilaco/miniconda3/lib/python3.5/site-packages/pygenometracks/tracksClass.py", line 246, in plot
    track.plot(plot_axis, chrom, start, end)
  File "/Users/favilaco/miniconda3/lib/python3.5/site-packages/pygenometracks/tracks/EpilogosTrack.py", line 95, in plot
    qcat_json = '{' + qcat_json.replace('id', '"id"').replace('qcat', '"qcat"') + '}'
AttributeError: 'numpy.float64' object has no attribute 'replace'
```

However, after re-doing the tabix command (with region chr10:12836000-12841000, only 5kb) to get the .qcat file, the following command works (PDF correctly generated):
`
pyGenomeTracks --tracks tracks.ini --region chr10:12830000-12841000 -o OUTPUT.pdf
`

LOG
```
INFO:pygenometracks.tracksClass:time initializing track(s):
INFO:pygenometracks.tracksClass:0.003542184829711914
DEBUG:pygenometracks.tracksClass:Figure size in cm is 40 x 4.0. Dpi is set to 72

INFO:pygenometracks.tracksClass:plotting 1. [x-axis]
/Users/favilaco/miniconda3/lib/python3.5/site-packages/matplotlib/cbook/__init__.py:424: MatplotlibDeprecationWarning:
Passing one of 'on', 'true', 'off', 'false' as a boolean is deprecated; use an actual boolean (True/False) instead.
  warn_deprecated("2.2", "Passing one of 'on', 'true', 'off', 'false' as a "
INFO:pygenometracks.tracksClass:plotting 2. [spacer]
INFO:pygenometracks.tracksClass:plotting 3. [input.qcat]
```

Could you look into this?

Thank you very much!

Francisco

**Comment by lldelisle** [comment]
Hi,
Sorry for the super late reply. Thank you very much for reporting this bug because this escaped our attention.
The bug was not linked to the size plotted but to the transition between these 2 lines:
```
chr10	12831801	12832001	id:1174740,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.009821,18], [0.1376,25], [0.1435,8] ]
chr10	12832200	12832400	id:1174741,qcat:[ [0,2], [0,24], [0,23], [0,22], [0,21], [0,20], [0,19], [0,18], [0,17], [0,16], [0,15], [0,14], [0,13], [0,12], [0,11], [0,10], [0,9], [0,7], [0,6], [0,5], [0,4], [0,3], [0,1], [0.07013,8], [0.202,25] ]
```
The region chr10:12832001-12832200 has no value, thus nan were assigned by the get_scores function but we forgot this case when we wrote the EpilogosTracks. I propose a bug fix in  #163.

**Comment by lldelisle** [comment]
The PR has been merged to the master branch. You can access the debugged version by 
```
pip install  git+https://github.com/deeptools/pyGenomeTracks.git
```
It will be part of the next release.
